templates/monarch-hand.hbs

```diff
@@ -23,7 +23,7 @@
 							</section>
 							<div class="card-controls">
 								{{#if @root.editable}}
-									<a class="card-control" title="{{localize 'CARD.Play'}}" data-action="play"><i
+									<a class="card-control" tabindex="0" title="{{localize 'CARD.Play'}}" data-action="play" aria-label="{{card.name}}, Play"><i
```

Comment:
Two questions here:
1. Is tab index 0 being applied to the play button of every card right? Or should the value increment or something?
2. Rationale for the label being "{name} , Play" instead of "Play {name}"?

As far as the code is concerned though:
1. At this point there are so many attributes on this element, some formatting should be done to reduce line-length
2. The label will need to be localized.

Reply:
1. Tab index should be 0 for all instances. 0 is a special case that simply adds an element into the tab order based on its position in the document, rather than manually ordering each element with a hardcoded index.
2. I'm not 100% confident on this, but I've noticed from listening to screen readers present information that the most common practice seems to be to provide information/identification first, then announce the interactions available. Stuff like, "Notifications, press to open" instead of "Open Notifications". I think this works better in an audio format where the user can't read ahead, because it cuts down the amount of time needed to skim through multiple elements. If the label is "Play {name}", then skimming a hand of cards sounds like, "Play Ace of Clubs, Play Ace of Hearts, Play 5 of Diamonds...", forcing the user to hear the superfluous "Play" repeatedly. By putting the name of the card first, however, the user can interrupt the narration by tabbing to the next element once they know the name of the card. It's a small time-save, but it adds up obviously, especially when they're looking for a specific card in a large hand.

I'll try to fix those two code concerns, if I can figure out the pull request editing function.

Reply:
No need to correct them, I can do so. I just have not had the time yet.

Reply:
> Tab index should be 0 for all instances. 0 is a special case that simply adds an element into the tab order based on its position in the document, rather than manually ordering each element with a hardcoded index.

Ah, I figured it might be something like that.

> I think this works better in an audio format where the user can't read ahead, because it cuts down the amount of time needed to skim through multiple elements. If the label is "Play {name}", then skimming a hand of cards sounds like, "Play Ace of Clubs, Play Ace of Hearts, Play 5 of Diamonds...", forcing the user to hear the superfluous "Play" repeatedly. By putting the name of the card first, however, the user can interrupt the narration by tabbing to the next element once they know the name of the card. 

I see, that does make sense. Cool.